.cmake-format.json

```diff
@@ -0,0 +1,40 @@
+{
+  "line_width": 80,
+  "tab_size": 2,
+  "max_subargs_per_line": 3,
+  "separate_ctrl_name_with_space": false,
+  "separate_fn_name_with_space": false,
+  "dangle_parens": false,
+  "bullet_char": "*",
+  "enum_char": ".",
+  "line_ending": "unix",
+  "command_case": "lower",
+  "keyword_case": "unchanged",
+  "additional_commands": {
+    "foo": {
```

Comment:
What's this command? :grinning:

Reply:
I believe it's the specific structure for custom cmake functions. So currently I don't think it is doing anything but it useful to keep in if we do want to add things in the future.